plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -220,3 +221,11 @@ def download_succeeded(self, report):
             if not added_unit.downloaded:
                 added_unit.downloaded = True
                 added_unit.save()
+
+            # the only reliable way to identify modular [S]RPMs is to analyze headers
+            # which are available only upon successful download.
+            if isinstance(unit, (models.RPM, models.SRPM)):
```

Comment:
I'm wondering, can a module have delta-RPMs? Can this flag be ever set on an SRPM? I thought it was being set during a (module) build process, would be nice to refer to some docs here.

Reply:
Flag can be set but there is no use for it at the moment, so I removed modular flag support for SRPMs.